rocky/rocky/views/ooi_view.py

```diff
@@ -146,7 +145,7 @@ def form_valid(self, form):
             form.add_error("__all__", str(exception))
             return self.form_invalid(form)
 
-    def get_success_url(self, ooi) -> str:
+    def success_url(self, ooi: OOI) -> str:
```

Comment:
I don't really like this. First of all it is a method, not a property, so I think `get_success_url` is the correct name. Secondly this will shadow the success_url property for everything that inherits from `FormView`. If the problem is that it conflicts with the already existing `get_success_url` I think `get_ooi_success_url` would be the better name.

Reply:
Ah I didn't know about this collision, I was trying to avoid the previous collision with `get_success_url`. Agreed that something along the lines of `get_ooi_success_url` would be better.

Reply:
:heavy_check_mark: